packages/web/src/pages/settings-page/components/desktop/ManagerMode/AccountsYouManageSettingsCard.tsx

```diff
@@ -21,22 +23,29 @@ const messages = {
 
 export const AccountsYouManageSettingsCard = () => {
   const [isModalOpen, setIsModalOpen] = useState(false)
+  const dispatch = useDispatch()
   const location = useLocation()
+  const isManagedAccountsRoute = doesMatchRoute(
```

Comment:
maybe worth memoizing this still? / keeping inside effects?

Reply:
done